docs/src/rust-feature-support.md

```diff
@@ -371,8 +371,8 @@ prefetch_read_data | No | |
 prefetch_read_instruction | No | |
 prefetch_write_data | No | |
 prefetch_write_instruction | No | |
-ptr_guaranteed_eq | Partial | |
-ptr_guaranteed_ne | Partial | |
+ptr_guaranteed_eq | Yes | Over-approximation: Returns nondet. value even if comparison is true |
+ptr_guaranteed_ne | Yes | Over-approximation: Returns nondet. value even if comparison is true |
```

Comment:
Should one of these say "false"

Reply:
No, these are not symmetric in the sense that if one returns true, the other one returns false. Under some circumstances, both may return false, which is the desired behavior. See https://doc.rust-lang.org/std/primitive.pointer.html#method.guaranteed_eq for more details.